testcafe/tests/photos/photos_start_upload_photos.js

```diff
@@ -23,11 +19,12 @@ fixture`Upload photos`.page`${TESTCAFE_PHOTOS_URL}/`
   })
 
 test('Uploading 1 pic from Photos view', async t => {
+  await t.maximizeWindow() //Real fullscren for VR
   ///there is no photos on page
   await page.initPhotoCountZero()
   await page.uploadPhotos([`${DATA_PATH}/${IMG0}`])
 
-  await t.fixtureCtx.vr.takeScreenshotAndUpload('Upload-1-pic.png')
+  await t.fixtureCtx.vr.takeScreenshotAndUpload('UploadImage/Upload-1-pic.png')
```

Comment:
How comes sometimes we do `if (t.fixtureCtx.isVR)` before taking a screenshot, and sometimes not?

Reply:
` if (t.fixtureCtx.isVR)`is use in methods that were written before using VisualReview, so we don't have to duplicate them (ie : `UploadFiles` is used in several tests : some already using VisualReview, and older tests which don't have screenshot yet, so we need a condition to know when to take screenshots)

Here, in `photos_start_upload_photos.js` we are in a `test` using VisualReview (as we do `await initVR(..)` in `.before` so we don't need to check the condition

Reply:
Ok I see, temporary code then. Thanks for the explanation!